plugin/src/main/java/net/dzikoysk/funnyguilds/feature/ban/BanUtils.java

```diff
@@ -50,7 +50,7 @@ public static String getBanMessage(User user) {
                 .map(ban -> {
                     FunnyFormatter formatter = new FunnyFormatter()
                             .register("{NEWLINE}", ChatColor.RESET + "\n")
-                            .register("{DATE}", messages.dateFormat.format(ban.getBanTime()))
+                            .register("{DATE}", messages.dateFormat.format(Date.from(ban.getTime())))
```

Comment:
`java.sql.Date` jest legacy. jak modern PR to używajmy modern api

Tu fajna tabelka :3

![image](https://user-images.githubusercontent.com/49173834/186652397-3cf68ddf-fc72-49d5-a42b-03e79b3854ea.png)